api/v1beta1/azuremanagedmachinepool_webhook.go

```diff
@@ -316,7 +317,8 @@ func (m *AzureManagedMachinePool) validateLastSystemNodePool(cli client.Client)
 		return nil
 	}
 
-	if ownerCluster.Spec.Paused {
```

Comment:
Does the `DeleteForMoveAnnotation` supercede this entirely? Wouldn't we still want to check for both conditions?

Reply:
Yes, we don't need this. Move annotation confirms that object is deleting because of clusterctl move operation.